Test.groovy

```diff
@@ -22,19 +22,37 @@ pipeline {
         sh "chmod u+x ./database-service/scripts/waitForService.sh && ./database-service/scripts/waitForService.sh localhost 9000"
       }
     }
-    stage('Test GraphQL engine') {
+    stage('Perform GraphQL engine tests') {
       steps {
         sh "docker-compose -f docker-compose-tests.yaml up hasura-service-tests"
       }
     }
+    // stage('Perform acceptance tests') {
+    //   steps {
+    //     sh "docker-compose -f docker-compose-tests.yaml up feature-tests"
+    //   }
+    // }
```

Comment:
When we have some acceptance tests ready, I enable this pipeline code. It makes no sense yet.